tools/releaseBuild/Images/microsoft_powershell_windowsservercore/PowerShellPackage.ps1

```diff
@@ -12,13 +12,16 @@ param (
     [string] $destination = "$env:WORKSPACE",
 
     [ValidateSet("win7-x64", "win7-x86", "win-arm", "win-arm64", "fxdependent", "fxdependent-win-desktop")]
-    [string]$Runtime = 'win7-x64',
+    [string] $Runtime = 'win7-x64',
+
+    [ValidateSet("min-size")]
+    [string] $Configuration = '',
```

Comment:
I would recommend renaming this variable. Other related build scripts use Configuration as per `dotnet build -c` for Release and Debug. Renaming this will reduce confusion.

Reply:
Maybe a switch like other build.psm1.

Reply:
The reason I made it a string parameter is that we may have other special `BuildConfiguration` in future. It may look confusing but we actually are using similar concept in `Start-PSBuild -Configuration`, which takes `CodeCoverage` as a legit value.

Reply:
I am fine with it being a string, but would prefer a rename. I believe we do not use the CodeCoverage anymore an should be removed in a separate PR.

Reply:
I decide to use a switch parameter for its simplicity. We can revisit this if there is a special build/packaging added in future.